packages/k6-tdk/src/client/user.ts

```diff
@@ -0,0 +1,102 @@
+import { check } from 'k6';
+import { RefinedResponse } from 'k6/http';
+
+import { Api } from '../api';
+import { Account } from '../auth';
+import { Version } from './client';
+
+
+export class User {
+  #api: Api;
+  #version: Version;
+  constructor(version: Version, api: Api) {
+    this.#version = version;
+    this.#api = api;
+  }
+
+  drives(): RefinedResponse<'text'> | undefined {
+    if (this.#version === Version.legacy) {
+      return;
+    }
+
+    const response = this.#api.graph.v1.me.drives();
+
+    check(response, {
+      'client -> dav.drives - status': ({ status }) => status === 200,
+    });
+
+    return response;
+  }
+
+  me(): RefinedResponse<'text'> | undefined {
+    if (this.#version === Version.legacy) {
+      return;
+    }
+
+    const response = this.#api.graph.v1.me.me();
+
+    if (!response) {
+      return;
+    }
+
+    check(response, {
+      'client -> user.me - status': ({ status }) => status === 200,
+    });
+
+    return response;
+  }
+
+  enable(id: string): RefinedResponse<'text'> | undefined {
+    if (this.#version === Version.latest) {
+      return;
+    }
+
+    const response = this.#api.ocs.v2.cloud.users.enable(id);
+
+    check(response, {
+      'client -> user.enable - status': ({ status }) => status === 200,
+    });
+
+    return response;
+  }
+
+  create(account: Account): RefinedResponse<'text'> {
+    let response;
+    switch (this.#version) {
+      case Version.latest:
+        response = this.#api.graph.v1.users.create(account);
+        break;
+      case Version.legacy:
+        response = this.#api.ocs.v2.cloud.users.create(account);
+        break;
+    }
+
+    check(response, {
+      'client -> user.create - status': ({ status }) => status === 200,
```

Comment:
maybe show status in log?
![image](https://user-images.githubusercontent.com/84779829/229107525-49f07b6b-efec-46de-85bc-01d58eaa4db7.png)

like:
 client -> user.create - status 200 -> green line
client -> user.create - status 400 -> red line